### ISSUE TITLE
[typescript-resolvers] immutableTypes=true generates Array instead of ReadonlyArray

### ISSUE THREAD

**Issue by jonaskello** [dotansimha/graphql-code-generator#3081]
**Describe the bug**
Using typescript-resolvers plugin to generate resolvers will generate code with `Array` instead of `ReadonlyArray` within `ResolversParentTypes` type.

**To Reproduce**

```yaml
config:
  immutableTypes: true
```

Generates this code:
```ts
/** Mapping between all available schema types and the resolvers parents */
export type ResolversParentTypes = {
  ProductTableRow: Omit<ProductTableRow, 'childRows'> & {childRows: Array<ResolversParentTypes['ProductTableRow']> },
}
```

Notice how `Array` is used in `childRows: Array<ResolversParentTypes['ProductTableRow']`.

**Expected behavior**
The generated code should be

```ts
/** Mapping between all available schema types and the resolvers parents */
export type ResolversParentTypes = {
  ProductTableRow: Omit<ProductTableRow, 'childRows'> & {childRows: ReadonlyArray<ResolversParentTypes['ProductTableRow']> },
}
```

**Environment:**

- OS: Macos
- `@graphql-codegen/...`: 1.9.1
- NodeJS: 12.6.0

**Comment by dotansimha** [comment]
@jonaskello Fixed in: https://github.com/dotansimha/graphql-code-generator/pull/3188

Can you please try the alpha? `1.9.2-alpha-572dfb0a.84`

**Comment by dotansimha** [comment]
Fixed in `v1.10.0`.
